__tests__/app.test.js

```diff
@@ -154,6 +155,51 @@ describe("GET /api/articles/:article_id/comments", () => {
     });
 })
 
+describe("POST /api/articles/:article_id/comments", () => {
+    test("status 201, responds with the newly added comment", () => {
+        return request(app)
+        .post("/api/articles/3/comments")
+        .send({
+            username: "icellusedkars",
+            body: "Not my cup of tea"
+        })
+        .expect(201)
+        .then((response) => {
+            expect(response.body.comment).toEqual({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: "icellusedkars",
+                body: "Not my cup of tea",
+                article_id: 3
+            });
+        });
+    });
```

Comment:
Could you add tests that check
- 400 an invalid `article_id` is provided `/api/articles/katherine/comments`
- 404 a valid, non-existent `article_id` is provided `/api/articles/10000/comments`